src/components/Balatro/Balatro.tsx

```
@@ -0,0 +1,237 @@
+    isRotate?: boolean;
+    mouseInteraction?: boolean;
+}
+
+function hexToVec4(hex: string): [number, number, number, number] {
+    const hexStr = hex.replace("#", "");
+    let r = 0,
+        g = 0,
+        b = 0,
+        a = 1;
+    if (hexStr.length === 6) {
+        r = parseInt(hexStr.slice(0, 2), 16) / 255;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)